addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15727,7 +15727,19 @@ msgctxt "#24163"
 msgid "Match Apple tvOS Standard (Siri remote)"
 msgstr ""
 
-#empty strings from id 24164 to 24990
+#. Header text for yes/no dialog about enable of broken addon
+#: xbmc/addons/GUIDialogAddonInfo.cpp
+msgctxt "#24164"
+msgid "Addon \"{0:s}\" broken"
+msgstr ""
+
+#. Yes/no dialog text with addon broken string value and to ask for use
+#: xbmc/addons/GUIDialogAddonInfo.cpp
+msgctxt "#24165"
+msgid "Addon marked as broken with following note:[CR][B][I][LIGHT]{0:s}[/LIGHT][/I][/B][CR][CR]Do you want to enable?"
```

Comment:
The drastic font change is a little jarring. I would just do italics and drop the bold. Italics results in a thinner, lighter font anyway, so `[LIGHT]` might not be required

Reply:
I thought I'd really like to highlight the text explicitly so that the user can quickly see what's wrong with it.
Using an add-on declared as broken can be dangerous.

Has rechecked now with all available skins, the `[LIGHT]` is really not needed and removed. The other two are seems good to have.

With **[I]{0:s}[/I]** only:
![Bildschirmfoto von 2020-08-13 04-46-34](https://user-images.githubusercontent.com/6879739/90088959-80ee1500-dd20-11ea-903a-b6e55cf6a8c6.png)

With **[LIGHT]{0:s}[/LIGHT]** only:
![Bildschirmfoto von 2020-08-13 04-48-10](https://user-images.githubusercontent.com/6879739/90088963-82b7d880-dd20-11ea-849c-ca14624d25b9.png)

With **[B]{0:s}[/B]** only:
![Bildschirmfoto von 2020-08-13 04-49-08](https://user-images.githubusercontent.com/6879739/90088969-864b5f80-dd20-11ea-80e9-62d15edef9be.png)

Reply:
Funny was also on all other skins can't seen that a broken one is enabled, only was there in addon dialog the button "Disable" present :rofl: